test/JDBC/upgrade/14_5/preparation/test_windows_login-vu-prepare.sql

```diff
@@ -0,0 +1,24 @@
+CREATE DATABASE ad_db;
```

Comment:
let's add common prepare file for all the version < 3.1

Reply:
Done!